### ISSUE TITLE
Deep Learning笔记——RNN&LSTM&GRU | Zessay's Blog

### ISSUE THREAD

**Issue by Zessay** [Zessay/Zessay.github.io#3]
https://zessay.github.io/posts/447687933.html 

 1. RNN1.1 RNN的前向传播 &emsp;设RNN从输入层到隐层的参数矩阵为$W_{xh}$，从上一个时间步到下一个时间步的隐层参数矩阵为$W_{hh}$，从当前时间步隐层结果到输出层结果的参数矩阵为$W_{hy}$；输出层的激活函数为$\text{softmax}$，损失函数为$\text{crossEntropy}$。于是得到下面公式：  $$ \begin{cases}a^{(t)}
